core/src/main/java/org/elasticsearch/index/query/EmptyQueryBuilder.java

```diff
@@ -34,10 +34,25 @@
 public final class EmptyQueryBuilder extends AbstractQueryBuilder<EmptyQueryBuilder> {
 
     public static final String NAME = "empty_query";
-
-    /** the one and only empty query builder */
     public static final EmptyQueryBuilder PROTOTYPE = new EmptyQueryBuilder();
```

Comment:
I kept this (and one more) because they were used a ton in tests and I figured I should handle that in another commit.